backend/baseclass.pm

```diff
@@ -221,7 +261,17 @@ sub do_capture ($self, $timeout = undef, $starttime = undef) {
                 # The maximum hits per 10 seconds saw on a half open socket was >100k
                 if (check_select_rate($buckets, $wait_time_limit, $hits_limit, fileno $fh)) {
                     my $console = $self->{current_console}->{testapi_console};
-                    OpenQA::Exception::ConsoleReadError->throw(error => "The console '$console' is not responding (half-open socket?). Make sure the console is reachable or disable stall detection on expected disconnects with '\$console->disable_vnc_stalls', for example in case of intended machine shutdown");
+                    my $fd_nr = fileno $fh;
+                    my $cnt = $buckets->{$fd_nr}->{cnt};
+                    my $name = $self->{select_read}->get_name($fh);
+                    my $msg = "The file descriptor $fd_nr ($name) hit the read attempts threashold of $hits_limit/${wait_time_limit}s by $cnt. ";
+                    $msg .= "Active console '$console' is not responding (half-open socket?). ";
```

Comment:
Didn't you say in the PR description

> And it must not be correlated to the current active console.

So this is possibly still misleading. Maybe add a "might" somewhere in the sentence or make it otherwise clear that it can also have other causes?

Reply:
ok like `Active console is '$console', it could be a be a half-open socket.` ?

Reply:
Actually I'm not a fan of this `half-open` socket thing anyway. Normally if a socket is clothed by the other end, you will realize it within the next read/write cycle.

Reply:
The full error message is now:
```
[2021-11-22T10:52:20.050854+01:00] [info] ::: backend::baseclass::die_handler: Backend process died, backend errors are reported below in the following lines:
  The file descriptor 13 (baseclass::cmdpipe) hit the read attempts threashold of 15000/30s by 32004. Active console 'root-virtio-terminal' is not responding, it could be a half-open socket or you need to increase _CHKSEL_RATE_HITS value. Make sure the console is reachable or disable stall detection on expected disconnects with '$console->disable_vnc_stalls', for example in case of intended machine shutdown.
   fd:13 (baseclass::cmdpipe) => 32004 age:30.00s
   fd:17 (qemu::start_qemu::qemu_pipe) => 1 age:58.13s
```

It looks like this
![Screenshot from 2021-11-22 10-57-54](https://user-images.githubusercontent.com/40127946/142841336-594a21a9-2620-4cec-ad73-ef51cca62951.png)

I didn't used newlines for each message part, as then this hint in the Web-UI does only show the first line!